include/linux/tee_drv.h

```diff
@@ -340,6 +340,28 @@ void *tee_get_drvdata(struct tee_device *teedev);
 struct tee_shm *tee_shm_alloc(struct tee_context *ctx, size_t size, u32 flags);
 struct tee_shm *tee_shm_alloc_kernel_buf(struct tee_context *ctx, size_t size);
 
+/**
+ * tee_shm_register_fd() - Register shared memory from file descriptor
+ *
+ * @ctx:	Context that allocates the shared memory
+ * @fd:		Shared memory file descriptor reference
+ *
+ * @returns a pointer to 'struct tee_shm'
```

Comment:
` * @returns a pointer to 'struct tee_shm' on success, or a errno on failure`

Reply:
From other inline description, could be
`* @returns a pointer to 'struct tee_shm' on success, and ERR_PTR on failure`